### PR TITLE
BAH-2403 | add. configured to have multiple buttons in registration extension point

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: ui/app/registration/views/patientcommon.html [hunk 1]
```diff
@@ -229,7 +241,7 @@
             <div class="identifier-ui"  ng-hide="!showExtIframe">
                 <div class="identifier-iframe">
                     <span class="close" ng-click="closeIdentifierPopup()">&times;</span>
-                    <iframe name="identifier-verification" id="identifier-popup" height="95%" width="100%"></iframe>
+                    <iframe id="ndhm-popup" height="95%" width="100%"></iframe>
```
Comment (gsluthra) [inline ui/app/registration/views/patientcommon.html:244]: The name NDHM/ABDM should not be hard coded anywhere in Bahmni product codebase. Can this pop-up be called: "identifer-extension-popup"?

Comment (SanoferSameera) [reply ui/app/registration/views/patientcommon.html]: done. its renamed

File: ui/app/registration/views/patientcommon.html [hunk 2]
```diff
@@ -170,9 +170,15 @@
                                    non-blank="identifier.identifierType.required && !identifier.hasIdentifierSources()"
                                    pattern-validate ng-change="identifier.generate()"
                                    ng-disabled="readOnlyExtraIdentifiers.indexOf(identifier.identifierType.name)>-2">
-                            <button class="verify-health-id-btn" ng-show="showIdentifierVerificationButton(identifier.identifierType.name,identifier.registrationNumber)" type="button"
-                                    ng-click="openIdentifierPopup(identifier.identifierType.name)">{{:: getDisplayName(identifier.identifierType.name)}}
-                            </button>
+                                <div class="verify-health-id-btn" ng-show="showIdentifierVerificationButton(identifier.identifierType.name,identifier.registrationNumber)">
+                                    <button type="button"
```
Comment (swatigogia2020) [inline ui/app/registration/views/patientcommon.html:174]: can we have a generic name instead of health-id

**Review by swatigogia2020 — APPROVED** [review]
Just a small name change

**Pull request merged**